driver-mct/cime_config/config_component_e3sm.xml

```diff
@@ -389,9 +391,11 @@
       <value compset="_MPASO">48</value>
       <value compset="_DLND.*_MALI">1</value>
       <value compset="_SLND.*SOCN.*_MALI">1</value>
+      <value compset="_DATM.*_SLND.*_MPAS.*_MALI">17520</value>
       <value compset="_DATM.*_SLND.*MPAS" grid="oi%oQU480">12</value>
       <value compset="_DATM.*_SLND.*MPAS" grid="oi%oQU240">12</value>
       <value compset="_DATM.*_SLND.*MPAS" grid="oi%oQU240wLI">12</value>
+      <value compset="_DATM.*_SLND.*MPAS.*_MALI" grid="oi%oQU240wLI">4380</value>
       <value compset="_DATM.*_SLND.*MPAS" grid="oi%oQU120">24</value>
```

Comment:
This one ("_DATM.*_SLND.*MPAS.*_MALI" grid="oi%oQU240wLI") and line 523 below are diff. (default coupling once every 2 hrs versus once every 30 mins) on purpose because of the low-res ocean mesh?